setup.py

```diff
@@ -73,6 +73,7 @@ def gftools_scripts():
         'protobuf',
         'requests',
         'tabulate',
+        'fs',
```

Comment:
if you don't import directly `fs` (aka pyfilesystem2), but you included it only to be able to import fontTools.ufoLib, I would recommend that you specify the fonttools requirement using the appropriate 'extra' notation, i.e. `fonttools[ufo]`, instead of only `fonttools`. This will make sure that the required extra requirements are met (including fs).